go/ql/src/Security/CWE-770/UncontrolledAllocationSize.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Slice memory allocation with excessive size value
+ * @description Allocating memory for slices with the built-in make function from user-controlled sources
+ *              can lead to a denial of service.
+ * @kind path-problem
+ * @problem.severity warning
+ * @security-severity 6.0
+ * @precision high
```

Comment:
I meant to ask if you calculated the security-severity and checked the other two of these make sense?